projects/novo-elements/src/elements/chips/ChipList.scss

```diff
@@ -7,20 +7,14 @@ $novo-chip-input-margin: 4px;
 
 .novo-chip-list {
   overflow: hidden;
-  flex-grow: 1;
 }
 
 .novo-chip-list-wrapper {
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
   align-items: center;
-  margin: -$novo-chip-list-margin;
-
-  input.novo-input-element,
-  .novo-chip {
-    margin: $novo-chip-list-margin;
-  }
+  gap: $novo-chip-list-margin;
```

Comment:
looks like this change bumped up the input text 4px, misaligning it with the text in other components. you can reproduce this in the query builder demo. we should try to fix this.
![4px margin](https://github.com/user-attachments/assets/ccbfd2f3-2d67-494b-9c4f-0ad6ebe5c0e5)

Reply:
Thanks for the attention to detail. I will take a look at this and find a clean fix.

This is mainly coming from an effort on my part to get away from a negative margins hack in the chip list that was having other effects. Hopefully we can keep going on it...

Reply:
fix looks good to me 👍